MAINTAINERS.md

```diff
@@ -3,7 +3,7 @@
 | Repository | Maintainers | Status |
 | --- | :-: | --: |
 | [.github](https://github.com/react-native-community/.github) | [@kelset](https://github.com/kelset) | active |
-| [apple-sign-in](https://github.com/react-native-community/apple-sign-in) | [@vonovak](https://github.com/vonovak), [@matt-oakes](https://github.com/matt-oakes), [@ericlewis](https://github.com/ericlewis) | starting |
+| [apple-sign-in](https://github.com/react-native-community/apple-sign-in) | XXX | archived |
```

Comment:
Just in case someone looks, perhaps worth it to link out to the two in-use implementations?

```suggestion
| [apple-sign-in](https://github.com/react-native-community/apple-sign-in) | XXX | archived - use [Expo](https://docs.expo.io/versions/latest/sdk/apple-authentication/) or [Invertase](https://github.com/invertase/react-native-apple-authentication) implementations |
```

Reply:
I like this idea!

Reply:
I agree with Mike, better to merge with the suggestion